tests/_util.py

```diff
@@ -20,10 +19,7 @@
 from gutenberg.acquire.metadata import set_metadata_cache
 import gutenberg.acquire.text
 
-if sys.version_info < (2, 7):
-    import unittest2 as unittest
-else:
-    import unittest
+import unittest
 assert unittest  # silence warning
```

Comment:
Nit: We can now even get rid of this import, the assert below and the indirect unittest imports in other places (like [here](https://github.com/hugovk/Gutenberg/blob/b42c65015914e15f36ae33dab8245c5849a987f1/tests/test_acquire.py#L18)).

Reply:
Done!